### ISSUE TITLE
[question] Where can I find gatsby wallpaper image?

### ISSUE THREAD

**Issue by RikoSmith** [gatsbyjs/gatsby#13337]
This is rather a simple question, but where can I find this image (image with toasts, jam jar, gatsby logo etc.). I'm preparing presentation for my course about Gatsby and would like to use it as a wallpaper. I've searched on repo, google images, gatsby official site and even in gatsby store - no result. If you have one, please, share the link)
![image](https://user-images.githubusercontent.com/24265041/56081944-b3d67f00-5e34-11e9-8348-7f880d436e78.png)
![image](https://user-images.githubusercontent.com/24265041/56081973-20ea1480-5e35-11e9-8d1b-b48507191f43.png)

**Comment by lannonbr** [comment]
I believe @jlengstorf may be able to help out with this

**Comment by delster** [comment]
Not the answer you're looking for, but I took that terminal screenshot, cropped the background to what was being tiled, and offer it here:

![tiled version](https://i.imgur.com/LVHcx1A.png)

Obviously this is a super low-res, terrible shadow of whatever the original designer can provide.. but it may work as a stop gap.

**Comment by jlengstorf** [comment]
Here’s the pattern. I set this to tile on my desktop.

![JAMstack Jammies Pattern](https://user-images.githubusercontent.com/163561/56143332-348c9c80-5f55-11e9-9ce0-6e16eb728618.png)

@marisamorby @fk would it make sense to put a “miscellaneous design” page together with things like the swag designs? Probably all of this needs an audit for brand from Flo since I definitely did _not_ follow brand guidelines when I designed this. 😅
